connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -60,16 +60,9 @@ message SQL {
   string query = 1;
 }
 
-// Relation that reads from a file / table or other data source. Does not have additional
-// inputs.
-message Read {
-  oneof read_type {
-    NamedTable named_table = 1;
```

Comment:
@cloud-fan, please see here, the Read had already had a union type and this just needs to be extended with an additional DataSource type.

Reply:
I see, yea we can add a new `DataSource` type here, in addition to the `NamedTable`. This seems like we give them a parent class `Read`. Any benefits of doing so compared to just flattening it? Please bear with me as I'm not very familiar with protobuf...